macros/generate_model_import_ctes.sql

```diff
@@ -0,0 +1,144 @@
+{% macro generate_model_import_ctes(model_name, leading_commas = false) %}
+
+    {%- if execute -%}
+    {%- set nodes = graph.nodes.values() -%}
+
+    {%- set model = (nodes
+        | selectattr('name', 'equalto', model_name) 
+        | selectattr('resource_type', 'equalto', 'model')
+        | list).pop() -%}
+
+    {%- set model_raw_sql = model.raw_sql -%}
```

Comment:
```suggestion
    {%- set model_raw_sql = model.raw_sql or model.raw_code -%}
```
This should work for dbt <=1.2 and >= 1.3